frontend/src/cljs/imas_seamap/events.cljs

```diff
@@ -470,4 +470,10 @@
   (assoc-in db [:display :seamap-drawer] true))
 
 (defn seamap-drawer-close [db _]
-  (assoc-in db [:display :seamap-drawer] false))
\ No newline at end of file
+  (assoc-in db [:display :seamap-drawer] false))
+
+(defn drawer-panels-open [db [_ panel props]]
+  (update-in db [:display :drawer-panels] #(conj % {:panel panel :props props})))
```

Comment:
(update-in db [:display :drawer-panels] conj {:panel panel :props props}))

Reply:
Thanks, I didn't realise that would work 👍